spritzle_cli/commands/stats.py

```diff
@@ -4,14 +4,14 @@
 from tabulate import tabulate
 
 
-@click.command('status', short_help='Show status of the session.')
+@click.command('stats', short_help='Show status of the session.')
```

Comment:
Maybe the help string should be updated to be accurate?

"Show session statistics."